stat/stat_unix1.go

```diff
@@ -0,0 +1,19 @@
+// +build linux solaris
+
+package stat
```

Comment:
i made this a separate package because when i removed the code duplication in the fs package due to this issue, i noticed that there was also code duplication in the content/local due to the same issue. so what would you like me to do?  i could:
- duplicate these two files into both the fs and content/local packages.
- leave the existing duplication in the content/local package alone and just include these two files in the fs package.